src/components/Question.js

```diff
@@ -97,13 +104,25 @@ class Question extends Component {
         url: baseUrl + '/q' + (index + 1),
       },
     };
-    let message;
 
+    let message;
     if (this.state.error) {
       message = 'Please select an answer';
     } else if (this.state.correctAnswer) {
       message = 'Correct!';
     } else if (this.state.correctAnswer === false) {
+      message = "Incorrect! Try Again!";
+    }
+
+    let explanation;
+    if (answerSubmitted) {
+      if (explanationRequested) {
+        explanation = <div className="explanation">{question.explanation}</div>
+      } else {
+      explanation = <button className="explanationButton" onClick={(event) => this.handleExplanationRequest()}>
+        See Explanation
+        </button>
+      }
```

Comment:
Maybe this logic could be in the return statement. Something like this:

```javascript
{answerSubmitted && explanationRequested && <div className="explanation">{question.explanation}</div>}
{answerSubmitted && !explanationRequested && <button className="explanationButton" onClick={(event) => this.handleExplanationRequest()}>See Explanation</button>}
```
I like trying to keep the JSX in the return statement as much as possible. In general, it is more readable this way.